ProcessLib/RichardsFlow/Tests.cmake

```diff
@@ -125,24 +125,44 @@ AddTest(
     EXECUTABLE_ARGS 1d-infiltration.prj
     TESTER vtkdiff
     DIFF_DATA
-    out_ex1_pcs_0_ts_0_t_0.000000_expected.vtu out_ex1_ts_0.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_50_t_2303.496783_expected.vtu out_ex1_ts_50.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_100_t_7303.496783_expected.vtu out_ex1_ts_100.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_150_t_12303.496783_expected.vtu out_ex1_ts_150.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_200_t_17303.496783_expected.vtu out_ex1_ts_200.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_250_t_22303.496783_expected.vtu out_ex1_ts_250.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_300_t_27303.496783_expected.vtu out_ex1_ts_300.vtu pressure pressure 1e-6 1e-6
-    out_ex1_pcs_0_ts_350_t_32303.496783_expected.vtu out_ex1_ts_350.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_400_t_37303.496783_expected.vtu out_ex1_ts_400.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_450_t_42303.496783_expected.vtu out_ex1_ts_450.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_500_t_47303.496783_expected.vtu out_ex1_ts_500.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_550_t_52303.496783_expected.vtu out_ex1_ts_550.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_600_t_57303.496783_expected.vtu out_ex1_ts_600.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_650_t_62293.496783_expected.vtu out_ex1_ts_650.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_700_t_67293.496783_expected.vtu out_ex1_ts_700.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_750_t_72293.496783_expected.vtu out_ex1_ts_750.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_800_t_77181.778033_expected.vtu out_ex1_ts_800.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_850_t_82181.778033_expected.vtu out_ex1_ts_850.vtu pressure pressure 1e-4 1e-5
-    out_ex1_pcs_0_ts_894_t_86400.000000_expected.vtu out_ex1_ts_894.vtu pressure pressure 1e-4 1e-5
+    out_ex1_ts_0_expected.vtu out_ex1_ts_0.vtu pressure pressure 1e-8 1e-8
```

Comment:
@waltherm I think the filename for the first argument is wrong, it should be `out_ex1_ts_0.vtu`. Full line would be:

```
out_ex1_ts_0.vtu out_ex1_ts_0.vtu pressure pressure 1e-8 1e-8
```

Reply:
You are right... I thought I had changed the reference file names...